app/src/main/java/com/wojdor/memolki/ui/feature/endgame/EndGameViewModel.kt

```
@@ -0,0 +1,19 @@
+
+@HiltViewModel
+class EndGameViewModel @Inject constructor(
+    savedStateHandle: SavedStateHandle
+) : MviViewModel<EndGameIntent, EndGameState>(
+    savedStateHandle,
+    EndGameState()
+) {
+
+    override fun onIntent(intent: EndGameIntent) {
+
+    }
```

detekt (1.23.8)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)